src/common/Parameter.cpp

```diff
@@ -1793,7 +1870,7 @@ bool Parameter::set_value_from_string( std::string s )
       return false;
    }
 
-   auto nv = std::atof(c);
+   float nv = std::atof(c);
```

Comment:
curious why you changed auto to float here?

Reply:
I had the same try-catch as for ints but then reverted it and forgot to reinstate `auto`.